SABnzbd.py

```diff
@@ -1086,9 +1086,24 @@ def main():
         if no_file_log:
             logging.info('Console logging only')
 
+    os.chdir(sabnzbd.DIR_PROG)
+
     logging.info('--------------------------------')
+    # If running from git repo, find and give info on branch and commit
+    try:
+        mygitcmd = 'git log -1 --oneline --decorate'.split(' ')
```

Comment:
Careful there, "git" may not be what you expect in the user's context. Consider find_on_path() or similar to build the cmd.

Do you really need more than just the revision? If not, you could go with `git rev-parse HEAD` instead. That would fit in nicely with existing code and thus also make the edit in interface.py redundant, which has the undesirable side effect of displaying a long commit hash in tagged revisions (i.e., releases).

Reply:
My goal is to on which branche I am, and which commit:

```
sander@brixit:~/git/sabnzbd$ git log -1 --oneline --decorate
640814d4 (HEAD -> feature/smarter-cache) Add debug logging when delaying the download

```

This is too little info for me:

```
sander@brixit:~/git/sabnzbd$ git rev-parse HEAD
640814d44c952675bed35b4f7df4d2f927bd3b5a
```